extension/src/cli/dvc/contract.ts

```diff
@@ -1,5 +1,3 @@
-import { Plot } from '../../plots/webview/contract'
-
 export const MIN_CLI_VERSION = '2.58.1'
```

Comment:
[I] Will need bumped

Reply:
Don't forget to remove the code that checks the CLI version when running `exp rename`

Reply:
Are these things that need to be done before merging?

Reply:
yep